153_save_sensor_data/sensor_data_store.py

```diff
@@ -65,40 +62,53 @@
 
 # Function Definitions
 
+# Initialize the sensor
+def initialize_sensor():
+    # From Carter's Script at Adafruit, it is recommended to start with a 
+    # frequency of 50 Khz due to "Clock Stretching".
+    i2c = busio.I2C(board.SCL, board.SDA, frequency=50000)
+    scd = adafruit_scd30.SCD30(i2c)
+    return scd
+
+# Create the directories to store the saved data
+def make_file_paths():
+    file_path = Path("scd_data/raw")
+    file_path.mkdir(parents=True,exist_ok=True)
+    file_path = Path("scd_data/interpolated")
+    file_path.mkdir(exist_ok=True)
```

Comment:
```suggestion
    Path("scd_data/raw").mkdir(parents=True, exist_ok=True)
    Path("scd_data/interpolated").mkdir(exist_ok=True)
```
This should be enough.